### ISSUE TITLE
HD SDI support/

### ISSUE THREAD

**Issue by lavidtech-1** [amiaopensource/vrecord#743]
Hi, 

I have been successfully capturing Digibeta through SDI.

Is there any support for HD SDI, I do not see it in the software?

What would be the steps to capture it if there is?

Again, thanks for all the help.

**Comment by retokromer** [comment]
Indeed, newer computers allow to do it. In the past, sometimes, the way via HuffYUV was chosen: firstly capture in HuffYUV and only as a second step transcode into FFV1.

**Comment by lavidtech-1** [comment]
Hi All,

Thanks for the response. I appreciate you taking the time to help with.

I am very new and am very unsure of the steps needed to register the format type in the decklink card.

Would you be able to provide some guidance on how to best do this?

Thank you in advance for any and all help.

**Comment by dericed** [comment]
Hi @lavidtech-1 could you try running these commands:

```
curl https://gist.githubusercontent.com/dericed/033c9c9ec7632f5d60fd0da169bcacab/raw/a4ef5acb53a8d7bacef447dc13104f290128e038/vrecord > vrecordhd
sudo chmod +x ./vrecordhd 
 ./vrecordhd
```

The first one downloads a patched version of vrecord, the second makes it executable and the third one runs it.

**Comment by Y1423Y** [comment]
Hi all , we also have some HD sources that need to be migrated. I just followed the steps that @dericed proposed and enter the patched version of vrecord, but it didn't work. The 'Input Utility' is ffmpeg, but the VIdeo Card can't  be found (the filed is blank). That doen't happened when I use the vrecord through the command '-vrecord'. In patched version I also noticed that in the page of 'Config' the field of 'Optional Tools', the light of 'bmdtools' turned red. What shoult I do to solve this problem? Thank you very much.

> Hi [@lavidtech-1](https://github.com/lavidtech-1) could you try running these commands:
> 
> ```
> curl https://gist.githubusercontent.com/dericed/033c9c9ec7632f5d60fd0da169bcacab/raw/a4ef5acb53a8d7bacef447dc13104f290128e038/vrecord > vrecordhd
> sudo chmod +x ./vrecordhd 
>  ./vrecordhd
> ```
> 
> The first one downloads a patched version of vrecord, the second makes it executable and the third one runs it.
